lms/djangoapps/teams/static/teams/templates/edit-team-member.underscore

```diff
@@ -0,0 +1,14 @@
+<li class="team-member">
+    <a class="member-profile" href="<%= memberProfileUrl %>">
+        <img class="image-url" src="<%= imageUrl %>" alt="profile page" />
```

Comment:
@efischer19 this looks like it will be part of a loop.  Can we prepend ```<%= username %>'s&nbsp;```  to the alt here so that to a screen reader user it appears as "efischer19's profile page"?  Otherwise when consumed out of context (like in a list of links on the page) they will get "x" number of identical links to "profile page"